specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1589 @@
+            "$ref": "./examples/Threads_DeleteChatThread.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ChatMessageReadReceipt": {
+      "description": "A chat message read receipt indicates the time a chat message was read by a recipient.",
+      "type": "object",
+      "properties": {
+        "senderId": {
+          "description": "Chat message read receipt sender id.",
+          "type": "string",
+          "readOnly": true,
+          "example": "8:acs:8540c0de-899f-5cce-acb5-3ec493af3800_0e59221d-0c1d-46ae-9544-c963ce56c10b"
+        },
+        "chatMessageId": {
+          "description": "Id for the chat message that has been read. This id is generated by the server.",
+          "type": "string",
+          "readOnly": true,
+          "example": "1591137790240"
+        },
+        "readOn": {
+          "format": "date-time",
+          "description": "Chat message read receipt timestamp. The timestamp is in ISO8601 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "readOnly": true,
+          "example": "2020-10-30T10:50:50Z"
+        }
+      }
+    },
+    "ChatMessageReadReceiptsCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of chat message read receipts.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ChatMessageReadReceipt"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "If there are more chat message read receipts that can be retrieved, the next link will be populated.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Error": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Description of the error",
+          "type": "string",
+          "readOnly": true
+        },
+        "target": {
+          "description": "If applicable, would be used to indicate the property causing the error",
+          "type": "string",
+          "readOnly": true
+        },
+        "innerErrors": {
+          "description": "If applicable, inner errors would be returned for more details on the error",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "SendReadReceiptRequest": {
+      "description": "Request payload for sending a read receipt.",
+      "required": [
+        "chatMessageId"
+      ],
+      "type": "object",
+      "properties": {
+        "chatMessageId": {
+          "description": "Id of the latest chat message read by the user.",
+          "type": "string",
+          "example": "1592435762364"
+        }
+      }
+    },
+    "ChatMessagePriority": {
```

Comment:
When are we planning to remove this? I talked to @ashwinder during public preview release, but it was too last-minute of a change. We agreed to hide this and when we have a more generalized approach bring it back. Ideally instead we should have a property bag that developers can add key-values. "Priority" by itself is too specific to be a property and does not scale.

Reply:
There is a feature we have not started yet for tagging which will have something like that. 
I have seeing 'Priority' in other vendors though, so not sure how expected could be to have an explicit priority. Good to loop some product folks ..